### PR TITLE
Force save changes & mild clean-up

### PR DESCRIPTION
**Pull request by Foxwells**
Forces anyone with deertaur/centaur body to have cloven/hoofed and 4 legs

Removes wolf arms, which is safe to remove since Wolf Pepper wasn't even working before I removed them.

Removes equine vag, since it literally just existed as an appearance definition and... nothing else.

### CODE REVIEW

File: classes/classes/Saves.as [hunk 1]
```diff
@@ -1792,18 +1792,28 @@ public function loadGameObject(saveData:Object, slot:String = "VOID"):void
 		player.hipRating = saveFile.data.hipRating;
 		player.buttRating = saveFile.data.buttRating;
 		
+		if (player.lowerBody > 3 && player.lowerBody < 5) { //Bitch's way of saying 4. So we can delete LOWER_BODY_TYPE_CENTAUR
```
Comment (Stadler76) [inline classes/classes/Saves.as]: Uhm ... that should read
```as3
		if (player.lowerBody == 4) {
```
then.
```
if (xyz > n - 1 && xyz < n + 1)
```
is kinda stupid, if xyz is an integer value.

File: classes/classes/Saves.as [hunk 2]
```diff
@@ -1792,18 +1792,28 @@ public function loadGameObject(saveData:Object, slot:String = "VOID"):void
...
+		
 		if (saveFile.data.legCount == undefined) {
 			if (player.lowerBody == LOWER_BODY_TYPE_DRIDER_LOWER_BODY) {
 				player.legCount = 8;
 			}
-			else if (player.lowerBody == LOWER_BODY_TYPE_CENTAUR) {
+			else if (player.lowerBody > 3 && player.lowerBody < 5) {
 				player.legCount = 4;
 				player.lowerBody = LOWER_BODY_TYPE_HOOFED;
 			}
```
Comment (Stadler76) [inline classes/classes/Saves.as]: No need, to repeat this here.

File: classes/classes/Saves.as (hunk 1 above)
Comment (Foxwells) [inline classes/classes/Saves.as]: If you notice, 4 isn't an entry in appearanceDefs anymore. I was skirting around directly saying 4 unless you want to throw errors.

Comment (Stadler76) [inline classes/classes/Saves.as]: Uhm ... so you're literally saying, that you can't use the **'value'** 4 in the whole codebase anymore?

You removed the constant (LOWER_BODY_TYPE_CENTAUR), not the value.

PS: I'd suggest to leave the constant there forever so no one gets the idea to reuse lowerBody = 4 later. Same goes for LOWER_BODY_TYPE_DEERTAUR

File: includes/appearanceDefs.as [hunk 3]
```diff
@@ -198,7 +198,6 @@ public static const LOWER_BODY_TYPE_HUMAN:int
 public static const LOWER_BODY_TYPE_HOOFED:int                                      =   1;
 public static const LOWER_BODY_TYPE_DOG:int                                         =   2;
 public static const LOWER_BODY_TYPE_NAGA:int                                        =   3;
-public static const LOWER_BODY_TYPE_CENTAUR:int                                     =   4; // DEPRECATED, use LOWER_BODY_TYPE_HOOFED and legCount=4
```
Comment (Stadler76) [inline includes/appearanceDefs.as]: Comment that line out, so the deprecation notice stays there,

File: includes/appearanceDefs.as [hunk 4]
```diff
@@ -218,7 +217,6 @@ public static const LOWER_BODY_TYPE_FERRET:int
 public static const LOWER_BODY_TYPE_CLOVEN_HOOFED:int                               =  21;
 //public static const LOWER_BODY_TYPE_RHINO:int                                       =  22;
 public static const LOWER_BODY_TYPE_ECHIDNA:int                                     =  23;
-public static const LOWER_BODY_TYPE_DEERTAUR:int                                    =  24; // DEPRECATED, use LOWER_BODY_TYPE_CLOVEN_HOOFED and legCount=4
```
Comment (Stadler76) [inline includes/appearanceDefs.as]: Comment that line out, so the deprecation notice stays there,

**Pull request merged**